index.js

```diff
@@ -57,5 +59,5 @@ module.exports = function(bytes, options){
     value = value.toFixed(config.digits);
   }
 
-  return sprintf(config.outputFormat, value, unit);
+  return sprintf(config.outputFormat, numberFormat.apply(numberFormat, [+value].concat(config.numberFormatOptions)), unit);
```

Comment:
Can you make this conditionally run numberformat if options were provided? Something like:

```
  if (config.numberFormatOptions) {
    value = numberFormat.apply( numberFormat, [value].concat(config.numberFormatOptions));
  }

  return sprintf(config.outputFormat, numberFormat.apply(numberFormat, [+value].concat(config.numberFormatOptions)), unit);
```